Readability upgrade.

ljy0812 opened the pull request:
(name) Duplicate--> IsDuplicateDataInVectorReturnIdx
(if) Readability upgrade

noopy0812 committed:
modify -(name) Duplicate--> IsDuplicateDataInVectorReturnIdx
(if) Readability upgrade

wnsgml972 commented on cppStudy/VectorManager.cpp:
```diff
@@ -26,39 +24,35 @@ namespace VectorSpace
+	bool VectorManager::DeleteDataInVector(int varId, int data)
 	{
-		if (!m_id2Vector[varId])
+		if (const auto& isExistData = m_id2Vector[varId]; !isExistData)
```
이거는 제 취향이긴 한데, Clean Code에 따르면 Data를 붙이는 명칭은 좋지 않다고 되어있습니다.
그 이유는 Person이라는 객체나 PersonData라는 객체를 구별할 수 없는 것처럼 Data의 명칭 자체가 모호하기 때문입니다.

ljy0812 replied:
오.. 감사합니다 반영하겠습니다

wnsgml972 replied:
넵 감사합니다!

wnsgml972 commented on cppStudy/main.cpp:
```diff
@@ -25,9 +25,9 @@ int main()
 	// Invalid
 	{
-		if (auto check = vectorManager->Append(4, 5); !check)
+	/*	if (auto check = vectorManager->Append(4, 5); !check)
```
이 부분 주석 처리하신 이유가 있나요?

ljy0812 replied:
어? 잘못한거 같습니다. 밑에꺼 풀었다가 멈춰서 위에꺼까지 다 주석처리한거 같아요 ㅎㅎ
다시 풀어놓겠습니다:)

wnsgml972 replied:
넵 감사합니다.

wnsgml972 reviewed (commented):
Issue를 먼저 달아서 Commit 시 Issue 번호를 (#1) 이런식으로 하셨으면 더 좋았을 것 같아요!

noopy0812 committed:
Upgrade readbility(#3)

change variable name(because word 'data' is vague)
delete comment(main function)
